memory-viz/src/tests/test_autofix.js

```diff
@@ -0,0 +1,3 @@
+console.log(
```

Comment:
This is good for testing purposes (and to test the change I suggested about permissions, above). However, after verifying that the autofix.ci commit works properly, you should delete this file before requesting another review.